### DISCUSSION TITLE
Videos always start in best quality

### DISCUSSION THREAD

**Discussion by Fmoment1** [robertodoering/harpy#695] [Ideas]
I went with a blank template instead of the bug one since this is more of an inconsistency than a bug.

We get an option to set the default quality of images, but the same choice has not been made available to videos. They always start in best quality and have to be manually set to normal or small using the settings icon at the bottom right of the video.

**Comment by robertodoering** [comment]
For suggestions feel free to open a discussion instead of an issue.

Currently videos don't have the option for a default quality because the lower quality videos are really bad and I would be surprised if anyone would want to lower the quality at all.
But I will think about adding an option for it. Thanks for the suggestion.

**Reply by ajmz** [reply]
> For suggestions feel free to open a discussion instead of an issue.
> 
> Currently videos don't have the option for a default quality because the lower quality videos are really bad and I would be surprised if anyone would want to lower the quality at all. But I will think about adding an option for it. Thanks for the suggestion.

I would also like to add video quality options, as some videos are more important about sound than picture quality, and I would like to add double speed or other different speed features like youtube.
